README.md

```diff
@@ -1,62 +1,48 @@
-[![Conclave SDK](.github/assets/logo.png)][conclave_website]
+[![Conclave Core SDK](.github/assets/logo.png)][conclave_website]
 
-## Overview
-[Conclave SDK][docs] is an open source platform that makes working with SGX enclaves easy.
+The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves very easily. You can 
```

Comment:
It's better to remove the 'very'.

The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves easily.